.travis.yml

```diff
@@ -1,3 +1,8 @@
 language: node_js
 node_js:
-  - "6"
\ No newline at end of file
+  - "6"
+install:
```

Comment:
It does this implicitly, but I like this change - I think it's a good idea to make this explicit as this is meant for beginners.